app.yaml

```diff
@@ -12,11 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-application: beacon-go
-version: 1
 runtime: go
 api_version: go1
 
 handlers:
 - url: /.*
   script: _go_app
+
+env_variables:
+  GOOGLE_CLOUD_PROJECT: project-id
```

Comment:
I'd leave this blank or change it to something like 'change-to-your-project-id'

Otherwise errors will be like:

"can't find table project-id/foo/bar" and it is less clear

Reply:
Even better would be to *not* set these by default [just comment them out] and throw an error or something on the request